traitsui/qt4/extra/progress_renderer.py

```diff
@@ -42,6 +42,9 @@ def paint(self, painter, option, index):
 
         # Draw it
         style = QtGui.QApplication.instance().style()
+        painter.save()
+        painter.translate(option.rect.left(), option.rect.top())
         style.drawControl(
             QtGui.QStyle.CE_ProgressBar, progress_bar_option, painter
         )
+        painter.restore()
```

Comment:
can you add comments here - with a reference to the link you mentioned in the PR? Without comments, we'll have to dig a bit to understand what the purpose of save/translate/restore are.